deployment/deployment.yaml

```diff
@@ -25,4 +25,15 @@ spec:
           requests:
             cpu: "2"
             memory: "2G"
+          livenessProbe:
```

Comment:
I think we need to assess what's the purpose of that `deployment` directory first. It seems redundant with the Kustomize manifests.

Reply:
Yes, it should be removed, but I see it's used in the README. We should update the later to rely on Kustomize manifests, and rely on `make deploy`.

Reply:
this suggestion has been added 👍